packages/kit/src/runtime/server/page/load_data.js

```diff
@@ -203,6 +203,15 @@ export function create_universal_fetch(event, state, fetched, csr, resolve_opts)
 	 */
 	return async (input, init) => {
 		const cloned_body = input instanceof Request && input.body ? input.clone().body : null;
+
+		/** @type {HeadersInit | undefined} */
+		let cloned_headers;
+		if (input instanceof Request && input.headers.keys().next().value?.length) {
+			cloned_headers = new Headers(input.headers);
+		} else {
+			cloned_headers = init?.headers;
+		}
```

Comment:
```suggestion
		const cloned_headers = input instanceof Request && [...input.headers].length ? new Headers(input.headers) : init?.headers;
```

we could do something like this to match up with `cloned_body` - it's unfortunate there is no size property on headers :/

Reply:
@ghostdevv sadly `pnpm lint` fails with this change, and I can't understand why:
```
[warn] src/runtime/server/page/load_data.js
[warn] Code style issues found in the above file. Forgot to run Prettier?
 ELIFECYCLE  Command failed with exit code 1.
```

Is there any way to see an error message? It's my 1st time with prettier

Reply:
The line's probably just too long. You can do a git pull, run format then push again.